test/csharp/TestOnnxRuntimeGenAIAPI.cs

```diff
@@ -5,10 +5,7 @@
 using System.IO;
 using Xunit;
 using Xunit.Abstractions;
-using Microsoft.ML.OnnxRuntimeGenAI;
-using System.Collections.Generic;
 using System.Linq;
-using System.Reflection.Emit;
```

Comment:
Most of the changes in this PR are now just style. I'd suggest undoing all of those. In the end, I think this is basically just about updating the TFMs and adding the project reference :smile: